internal/gok/vmrun.go

```diff
@@ -154,6 +156,11 @@ func (r *vmRunConfig) runQEMU(ctx context.Context, fullDiskImage string) error {
 		qemuBin = "qemu-system-aarch64"
 	}
 
+	netdev := "user,id=net0,hostfwd=tcp::8080-:80,hostfwd=tcp::8022-:22"
+	if r.netdev != "" {
+		netdev += "," + r.netdev
+	}
```

Comment:
Should we completely override the netdev rather than adding it to the default one? Otherwise we can never remove the default mapping, which in some cases might fail due to conflict and render the whole `run` command to fail.

If the user wants to extend the current one they can always copy the default and extend and then pass everything it.

Reply:
Changed. This will however no longer allow gok to react to e.g. `HttpUpdate` port.

Reply:
We can make gok react to a changed HttpUpdate port later by changing the default to be empty and then filling it in later when empty. I think this is good enough for now.